subgraphs/smolverse/src/helpers/collections.ts

```diff
@@ -0,0 +1,31 @@
+import { Address, TypedMap } from "@graphprotocol/graph-ts";
+import {
+  SMOL_BODIES_ADDRESS,
+  SMOL_BODIES_PETS_ADDRESS,
+  SMOL_BRAINS_ADDRESS,
+  SMOL_BRAINS_PETS_ADDRESS
+} from "@treasure/constants";
+import { log } from "matchstick-as";
```

Comment:
```suggestion
import { Address, TypedMap, log } from "@graphprotocol/graph-ts";
import {
  SMOL_BODIES_ADDRESS,
  SMOL_BODIES_PETS_ADDRESS,
  SMOL_BRAINS_ADDRESS,
  SMOL_BRAINS_PETS_ADDRESS
} from "@treasure/constants";
```

Fixed the import.

Reply:
oof..thank you! 🙏 I found a few other places where I was importing from `matchstick-as` instead and resolved those as well